docker/install_airflow.sh

```diff
@@ -11,6 +11,13 @@ PIPOPTS=""
 #export SLUGIFY_USES_TEXT_UNIDECODE=no
 export AIRFLOW_GPL_UNIDECODE=yes
 
+echo "** Installing numpy."
+pip $PIPOPTS install numpy
```

Comment:
Is numpy a requirement for Airflow?

Reply:
Airflow requires Pandas which in-turn installs numpy. There is a version mismatch happening with numpy. So, if we are installing numpy before installing Pandas, it is forcing the right version of numpy to be installed.

Reply:
What version mismatch? Pandas would define which version of numpy it's compatible with in a stetup.py or requirments.txt file and PIP should be installing the latest available version that's compatible with Panads.

Reply:
I was using Python 2.7.9 and was trying to install "apache-airflow" package. This packages tries to install Pandas 0.24.2 and Pandas in-turn installs Numpy 1.17.0. This is throwing error saying that Python 3.5 is required. This was solved by installing Numpy prior to the Airflow package installation